homework-submission/my-project/src/todo.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const todos = [
+    { id: 1, description: 'Do my dutch homework', deadline: 'FRI 19 SEP' },
+    { id: 2, description: 'React homework', deadline: 'SAT 20 SEP' },
+    { id: 3, description: 'preaper to React week2 ', deadline: 'SUN 21 SEP' }
+
+]
+
+class TodosList extends React.Component {
+    item = todos.map((item) => (
+        <li >
+            {item.id + "- " + item.description + " DeadLine: " + item.deadline}
```

Comment:
- You could have another component for each `li`
- the list is already odered list so we don't need `item.id`, laso don't use item.id for indexing but the index which map return. here you called it `i`

- You could use template literal here